core/controllers/resources.py

```diff
@@ -74,6 +74,8 @@ def get(self, exploration_id, encoded_filepath):
             fs = fs_domain.AbstractFileSystem(
                 fs_domain.ExplorationFileSystem(exploration_id))
             raw = fs.get(filepath)
+
+            self.response.cache_control = 'public, max-age=600'
```

Comment:
Do we need to do something like https://groups.google.com/forum/#!topic/google-appengine-python/OCCOJNGTx34 to ensure it's caching properly?